pkg/cli/doctor.go

```diff
@@ -212,16 +212,16 @@ func fromCluster(
 	}
 	stmt := `
 SELECT id, descriptor, crdb_internal_mvcc_timestamp AS mod_time_logical
-FROM system.descriptor ORDER BY id`
-	checkColumnExistsStmt := "SELECT crdb_internal_mvcc_timestamp FROM system.descriptor LIMIT 1"
+FROM system.descriptor  AS OF SYSTEM TIME '-0.1s' ORDER BY id`
```

Comment:
Good point, and we should use a constant here. The only painful thing here is that from a unit testing viewpoint, AOST queries with longer intervals will mean that we need to modify our tests to sleep/wait for that long. Otherwise, we might have a time travel query before the existence of any tables.

Reply:
Ah, right. Can we have this be a global var which gets overridden by an `envutil.EnvOrDefault...` using an env var which is only set in test settings, or something? Presumably there's machinery which exists to do this kind of thing, though I'm not familiar with it. Perhaps @tbg or @knz can enlighten us?

A fraction of a second makes sense for tests. Perhaps this is also fine in prod, idk, in any case it's better than what we presently have, I suppose.

Reply:
Can either of these have default values which are different depending on whether we're running tests or not?

Reply:
oh I think NO, but sorry that didn't make the point clear. I was wondering if it's possible that users would want to change the setting. But the odds is pretty low. And for testing, I we could just set the setting while a db handle is created, but if we could achieve from env var then I agree that's better.

Reply:
I definitely agree with you that they might want to, and perhaps even disable the AOST thing altogether.

Reply:
I went with the buildutil flag to distinguish the cases.